src/pfe/file-watcher/server/test/functional-test/lib/project.ts

```diff
@@ -74,3 +74,7 @@ export async function checkNewLogFile(projectID: string, type: string): Promise<
 export async function updateProjectForNewChange(projectID: string, timestamp: number,  chunkNum: number, chunk_total: number, eventArray: projectEventsController.IFileChangeEvent[]): Promise<projectEventsController.IUpdateProjectSuccess | projectEventsController.IUpdateProjectFailure> {
   return await filewatcher.updateProjectForNewChange(projectID, timestamp, chunkNum, chunk_total, eventArray);
 }
+
```

Comment:
why not use the function from your utils below?

Reply:
Which function?

Reply:
`functional-test/lib/utils.removeProjectFromRunningBuild`

Reply:
But use it where?

Reply:
sorry, i commented on a blank line, the next line in project.ts is function `project.removeProjectFromRunningBuildQueue` 

was thinking, if the function `functional-test/lib/utils.removeProjectFromRunningBuild` is the doing the same thing

Reply:
Oh, ok so the `project.removeProjectFromRunningBuildQueue` is in the source code. We import it into the test code and then use it inside another function https://github.com/ssh24/codewind/blob/992820388598f41c3d3a883b5750b14103fefb2b/src/pfe/file-watcher/server/test/functional-test/lib/project.ts#L79